activate_dev_from_wheel.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Python environment name
+PYTHON_ENV_NAME=$1
+
+if [ -z "${PYTHON_ENV_NAME}" ]; then
+    echo "Error: Please provide a name for the Python virtual environment"
+else
+    export CATALYST_PYTHON_ENV_DIR=/tmp/$PYTHON_ENV_NAME
+
```

Comment:
Should we also install `requirements.txt` into the new environment, since it contains the packages needed to run tests etc?

Reply:
I think doing this first will also prevent pip from installing testing versions of common dependency packages like `urllib` etc. From my testing this is also fixes the warning I reported above :)